src/frontend/js/settings/index.ts

```diff
@@ -0,0 +1,33 @@
+import { isArray, mergeWith } from 'lodash-es';
+import prodSettings from './settings.prod';
+
+let localSettings;
+try {
+  localSettings = require('./settings.local.ts');
```

Comment:
I liked the term `dev` over `local`

Reply:
I choose local because i thought that it could be used in some production instances, if we get some spécific configuration per instance (like scss theme  ?)

Reply:
Not quit sure to understand. Furthermore, we are merging this settings only when we are not in a production environment.

Reply:
i'll rename.
But it feel like these settings are local, it can be for dev usage or for deployment usage.